src/main/java/org/jitsi/videobridge/LipSyncHack.java

```diff
@@ -210,17 +211,37 @@ public void onRTPTranslatorWillWriteAudio(
             return;
         }
 
+        SimulcastReceiver recv = sourceVC.getTransformEngine()
+            .getSimulcastEngine().getSimulcastReceiver();
+
+        Long receiveVideoSSRC;
+        if (recv != null && recv.isSimulcastSignaled())
+        {
+            // FIXME this is a little ugly
+            receiveVideoSSRC = sourceVC.getTransformEngine()
```

Comment:
Thanks for the suggestion Pawel, I really appreciate it and I agree with you that this looks ugly. I plan to simplify it like this https://github.com/jitsi/jitsi-videobridge/blob/cc/src/main/java/org/jitsi/videobridge/LipSyncHack.java#L213 (please do not pay attention to the unchecked array accesses as the code is not finished yet).